tests/src/actionContainers/ActionProxyContainerTests.scala

```diff
@@ -151,6 +155,25 @@ class ActionProxyContainerTests extends BasicActionRunnerTests with WskActorSyst
         })
     }
 
+    it should "extract and run a compatible tgz exec" in {
+        val tgz = FileUtils.readFileToByteArray(new File(TestUtils.getTestActionFilename("blackbox.tgz")))
```

Comment:
for general black box containers, the archive format is up to the user - it need not be either gzip or zip; the only constraint is that it fits without our set limits.

for these native activation containers, since the binary must be built inside a compatible container (namely the whisk/dockerskeleton), i see less value in adding zip support - there should be just one format for ease of maintenance.